### PR TITLE
misc kernel and system patches

### PR DESCRIPTION
**Pull request by etbe**

### CODE REVIEW

File: policy/modules/admin/usermanage.te [hunk 1]
```diff
@@ -69,7 +69,7 @@ role useradd_roles types useradd_t;
 # Chfn local policy
 #
 
-allow chfn_t self:capability { chown dac_override fsetid setgid setuid sys_resource };
+allow chfn_t self:capability { chown dac_override fsetid setgid setuid sys_ptrace sys_resource };
```
Comment (dsugar100) [inline policy/modules/admin/usermanage.te:72]: sys_ptrace doesn't seem correct here.  Why does chfn_t need to ptrace?

Comment (etbe) [reply policy/modules/admin/usermanage.te:72]: Not sure and I can't reproduce it so I'll remove it.

**Review by pebenito — CHANGES_REQUESTED** [review]
Please revise the PR as I described in #711.

**Comment by etbe** [comment]
I'll do new smaller PRs for ths.

**Pull request closed without merging**